File: src/app/model/taikhoan.ts
export class tk {
    _id: number;
    username: string;
    password: string;
    tennv: string;
    email: string;
    hoatdong: boolean;
    giayphep: boolean;
    _id_group: number;
    constructor(_id: number, username: string, password: string, tennv: string, email: string, hoatdong: boolean, giayphep: boolean, _id_group: number) {
        this._id = _id;
        this.username = username;
        this.password = password;
        this.tennv = tennv;
        this.email = email;
        this.hoatdong = hoatdong;
        this.giayphep = giayphep;
        this._id_group = _id_group;
    }
}

File: dienmayxanhapi/dienmayxanhapi/Model/taikhoan.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace dienmayxanhapi.Model
{
  public class taikhoan
  {
    public int? _id { get; set; }
    public string username { get; set; }
    public string password { get; set; }
    public string tennv { get; set; }
    public string email { get; set; }
    public Boolean hoatdong { get; set; }
    public Boolean giayphep { get; set; }
    public int? _id_group { get; set; }
  }
}
